cmd/pod-scaler/testdata/zz_fixture_TestMutatePodResources_resources_to_add.diff

```diff
@@ -76,7 +76,7 @@
   		},
   		EphemeralContainers: nil,
   		RestartPolicy:       "",
-  		... // 30 identical fields
+  		... // 31 identical fields
```

Comment:
The unit tests compare the diff.
The expected diff is still there.
1 More identical field should come from API change.